changelogs/fragments/unset-rhui-release.yml

```diff
@@ -0,0 +1,5 @@
+---
+major_changes:
```

Comment:
I don't see this as a major change

Reply:
@djdanielsson thanks for taking a look. I listed it as a major change since it will change the behavior for the RHUI upgrade type. Before this change, the version lock was not unset, and after this change it will be. But if the team wants to change it to a minor_change I'm fine with that.